tools/admin/wskadmin

```diff
@@ -101,7 +101,7 @@ def parseArgs():
     subparsers.required = True
 
     propmenu = subparsers.add_parser('user', help='manage users')
-    propmenu.add_argument('-w', '--view', help='the subject view to query', default='subjects')
+    propmenu.add_argument('-w', '--view', help='the subject view to query', default='subjects.v2.0.0')
```

Comment:
this is introduced to allow for the older view to still be used. 

`wskdamin-next` requires no update because it uses the configuration files. @chetanmeh please confirm my understanding.

Reply:
Thats correct